src/components/toolbar/LevelSelector.tsx

```diff
@@ -410,13 +410,13 @@ const buildOptions = (
           isDisabled: sectionDepth <= 1 && sectionOffset <= 1,
         }),
         buildOption({
-          nodeType: nodes.ordered_list,
+          nodeType: nodes.list,
```

Comment:
This should have a list type.

Reply:
this will always be disabled, but I have added the list type to fetch the correct icon